scripts/run_integration_tests_aarch64.sh

```diff
@@ -214,7 +214,7 @@ if [ $RES -eq 0 ]; then
 
     newgrp kvm << EOF
 export RUST_BACKTRACE=1
-time cargo test --no-default-features --features "integration_tests,mmio,kvm" "tests::parallel::test_aarch64_pe_boot" -- --nocapture
+time cargo test --no-default-features --features "integration_tests,mmio,kvm" "tests::parallel::test_initramfs" -- --nocapture
```

Comment:
This change is just to verify the functionality working.
Another PR https://github.com/cloud-hypervisor/cloud-hypervisor/pull/1463 is fixing this.